go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/infobloxopen/atlas-app-toolkit
+
+go 1.14
+
+require (
+	contrib.go.opencensus.io/exporter/ocagent v0.7.0
+	github.com/DATA-DOG/go-sqlmock v1.4.1
+	github.com/dgrijalva/jwt-go v3.2.1-0.20200107013213-dc14462fd587+incompatible
+	github.com/golang/protobuf v1.4.2
+	github.com/google/uuid v1.1.1
+	github.com/grpc-ecosystem/go-grpc-middleware v1.2.0
+	github.com/grpc-ecosystem/grpc-gateway v1.14.6
+	github.com/jinzhu/gorm v1.9.12
+	github.com/jinzhu/inflection v1.0.0
+	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/lib/pq v1.3.1-0.20200116171513-9eb3fc897d6f
+	github.com/sirupsen/logrus v1.4.2
+	github.com/stretchr/testify v1.5.1
+	go.opencensus.io v0.22.3
+	golang.org/x/net v0.0.0-20200520182314-0ba52f642ac2
+	google.golang.org/api v0.26.0 // indirect
+	google.golang.org/genproto v0.0.0-20200527145253-8367513e4ece
+	google.golang.org/grpc v1.29.1
+	gopkg.in/yaml.v2 v2.2.8 // indirect
+)
+
+replace (
```

Comment:
The following packages are locked according to versions from Gopkg.lock. `Go mod tidy` tries to upgrade the mentioned packages to newer versions, although, these versions aren't major, looks like they contain breaking changes.
As result, TestForwardResponseMessageWithNil test fails with the following error message:
```
--- FAIL: TestForwardResponseMessageWithNil (0.00s)
panic: interface conversion: interface {} is nil, not map[string]interface {} [recovered]
	panic: interface conversion: interface {} is nil, not map[string]interface {}

goroutine 62 [running]:
testing.tRunner.func1.1(0x936f20, 0xc000324450)
	/usr/local/go/src/testing/testing.go:1072 +0x30d
```
https://github.com/infobloxopen/atlas-app-toolkit/blob/55b3a20b5562a04274a9e6cb8e58a432e398a993/gateway/response_test.go#L271

So, if we want to upgrade to the latest version, we need to investigate the potential impact on our codebase.